### PR TITLE
xgov-192-PiPhi Network-Smart Home Visualization Platform

### PR DESCRIPTION
**Pull request by KelvinSan**

### CODE REVIEW

File: Proposals/xgov-188.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+id: 188
```
Comment (nonfungibleab) [inline Proposals/xgov-188.md:2]: ID must be 192 to match PR https://github.com/algorandfoundation/xGov/pull/192/files

File: Proposals/xgov-188.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+  
+
+Employee 1 ($75,000/Yr): Responsible for customer service, resolving support tickets, and development. 
+
+  
+
+Employee 2 ($40,000): Responsible for customer service and resolving support tickets. 
+
+
```
Comment (nonfungibleab) [inline Proposals/xgov-188.md:151]: Grants are only paid when all is delivered. [Please read ARC-34](https://arc.algorand.foundation/ARCs/arc-0034) to ensure you understand the rules.

File: Proposals/xgov-188.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+ 
+
+All members have worked together since PiPhi’s inception. 
+
+## Experience with Algorand
+The evolution into PiPhi’s current state traces its origins back to October of 2021. During this time, one of the cofounders developed the first smart contract in the Algorand Test Net to test the viability of Algorand as the mechanism to allow PiPhi to execute its vision. Even in the early stages of this project, Algorand impressed all founding members of this team with its ease of use and seamless user development experience. During PiPhi’s embryonic stages, the founders continued to repeat a single refrain that captured this seamless user experience: “It just works.” To date, PiPhi through the Algorand blockchain has performed over 470,000,000 transactions and we’re firm believers that Algorand “just works.”  
+
+## Present Proposal
+
```
Comment (nonfungibleab) [inline Proposals/xgov-188.md:53]: Please clearly state the deliverables of your proposal and how the community can verify completion.

**Comment by nonfungibleab** [comment]
The xGov program aims to provide funding for individuals or teams to:
Develop of **open source applications and tools** (eg. an open source AMM or contributing content to an Algorand software library).
Develop Algorand education resources, preferably in languages where the resources are not yet available(eg. a video series teaching developers about Algorand in Portuguese or Indonesian).

**Comment by KelvinSan** [comment]
@nonfungibleab I think all suggested changes have been made. Please let us know if any other changes are required, The open source aspect of this project is a part of the proposal. Thank you for reviewing our submission

File: Proposals/xgov-192.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
+---
+id: 192
+period: 4
+title: PiPhi Network Smart Home Visualization Platform
+author: Justin Somerville <jsomerville@piphi.network>, Kelvin Sanyaolu (@KelvinSan)
+email: jsomerville@piphi.network
+discussions-to: https://forum.algorand.org/t/xgov-188-piphi-network-submission/11807
+company_name: PiPhi Network
+category: Other
+focus_area: Environmental Monitoring, Software, Developers
```
Comment (SudoWeezy) [inline Proposals/xgov-192.md:10]: Please refer to the error below.

 | focus_area: Environmental Monitoring, Software, Developers
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ must be one of: `Banking`, `Defi`, `Dex`, `Gaming`, `Identity`, `Marketplace`, `Metaverse`, `NFT`, `Oracle`, `Storage`, `User Onboarding`, `IDE`, `Teal`, `Deployment`, `Libraries`, `Monitoring`, `Node`, `Education`, `Social`, `Wallet`, `Other.`

**Review by SudoWeezy — CHANGES_REQUESTED** [review]

File: Proposals/xgov-192.md (hunk 4 above)
Comment (KelvinSan) [reply Proposals/xgov-192.md]: Change has been made

Comment (KelvinSan) [reply Proposals/xgov-192.md]: @SudoWeezy Changes have been made, please let us know if there is anything else we need to fix. Thank you

**Comment by KelvinSan** [comment]
@SudoWeezy @nonfungibleab All changes have been made, how can we initialize the workflow approval ?

**Review by SudoWeezy — APPROVED** [review]

**Pull request merged**